blackbox/suites/drive9/local_overlay_build/module.py

```
@@ -125,6 +126,13 @@ def base_repo_env(self, ctx: Context) -> dict[str, str]:
         for key, path in values.items():
             path.mkdir(parents=True, exist_ok=True)
             env[key] = str(path)
+        # Ensure a Node version that satisfies kimi-code's engine pin.
+        try:
+            node_bin = ctx.deps.ensure_node_version(">=24.15.0")
+            node_bin_dir = str(Path(node_bin).resolve().parent)
+            env["PATH"] = f"{node_bin_dir}:{env.get('PATH', '')}"
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)